packages/scene-composer/src/components/panels/scene-components/tag-style/ColorPicker/ColorPicker.tsx

```diff
@@ -24,14 +24,15 @@ export const ColorPicker = ({
   onUpdateCustomColors,
   colorPickerLabel,
   customColorLabel,
-}: IColorPickerProps): JSX.Element => {
+}: IColorPickerProps) => {
   const [showPicker, setShowPicker] = useState<boolean>(false);
   const [newColor, setNewColor] = useState<string>(color);
   const [showChromePicker, setShowChromePicker] = useState<boolean>(false);
   const [hexCodeError, setHexCodeError] = useState<string>(''); // State variable for hex code error
   const [customInternalColors, setCustomInternalColors] = useState<string[]>(customColors ?? []);
+  const generateRandomString = Math.random().toString(16).slice(2);
+  const [randomDomId] = useState<string>(generateRandomString);
```

Comment:
Why are you using `useState` here? You never update this state.

Reply:
Because I did not want to change the ID of the UI elements for every render but I wanted to use a unique one for different instances of color picker. State variable won't change for every render unless we update it with set state.